spec/quotas/quota.mdown

```diff
@@ -4,45 +4,45 @@ The quota is an object that displays the limit set to an account usage as well a
+* **resourceType**: "`ResourceType`" The resource type of the quota.
 
-* **used**: `UnsignedInt` The current usage of the defined quota. Computation of this value is handled by the server.
+* **used**: "`UnsignedInt`" The current usage of the defined quota. Computation of this value is handled by the server.
 
-* **limit**: `UnsignedInt` The hard limit set by this quota. Objects in scope may not be created or updated if we reach this limit. It should be higher than the `warnLimit` and the `softLimit`.
+* **limit**: "`UnsignedInt`" The hard limit set by this quota. Objects in scope may not be created or updated if we reach this limit. It should be higher than the `warnLimit` and the `softLimit`.
 
-* **scope**: `Scope` The `Scope` of this quota.
+* **scope**: "`Scope`" The `Scope` of this quota.
 
-* **name**: `String` The name of the quota object. Useful for managing quotas and use queries for searching.
+* **name**: "`String`" The name of the quota object. Useful for managing quotas and using queries for searching.
 
-* **datatypes**: `String[]` A list of all the data types values that are applying to this quota. This allows to assign
+* **dataTypes**: "`String[]`" A list of all the data types values that are applying to this quota. This allows to assign
 quotas to separated or shared data types. This MAY include data types the client does not recognise. Clients MUST
 ignore any unknown data type in the list.
 
-The quota object MAY contain the following field:
+The quota object MAY contain the following fields:
 
-* **warnLimit**: `UnsignedInt|null` The warn limit set by this quota object. It can be used to send a warning to an
+* **warnLimit**: "`UnsignedInt|null`" The warn limit set by this quota object. It can be used to send a warning to an
 entity about to reach the hard limit soon, but with no action taken yet. If set, it should be lower than the
 `softLimit` and the `limit`.
 
-* **softLimit**: `UnsignedInt|null` The soft limit set by this quota object. It can be used to still allow some operations,
-but refusing some others. What is allowed or not is up to the server. For example, it could be used for blocking outgoing
+* **softLimit**: "`UnsignedInt|null`" The soft limit set by this quota object. It can be used to still allow some operations,
+but refuse some others. What is allowed or not is up to the server. For example, it could be used for blocking outgoing
 events of an entity (sending emails, creating calendar events, ...) while still receiving incoming events (receiving emails,
 receiving calendars events, ...). If set, it should be higher than the `warnLimit` but lower than the `limit`.
 
-* **description**: `String|null` Arbitrary free, human readable, description of this quota. Might be used to explain
+* **description**: "`String|null`" Arbitrary free, human readable, description of this quota. Might be used to explain
 where the limit comes from and explain the entities and data types this quota applies to.
```

Comment:
How is handled internationalisation? (CF recent review comments)

Reply:
Wasn't fixed yet :) should be now